templates/named.conf.options.erb

```diff
@@ -62,6 +62,12 @@ check-names slave <%= @check_names_slave %>;
 check-names response <%= @check_names_response %>;
 <% end -%>
 
+<% if @statistic_channel_ip and @statistic_channel_ip -%>
```

Comment:
This needs a test.

Can you check out the spec/ folder and find similar tests?

I would like to see that the inet bla lport bla line is not in there by default, but is in there when you set the right ip/port. I wouldn't bother testing validation.